gcp_variant_transforms/beam_io/vcfio.py

```diff
@@ -715,3 +847,56 @@ def __init__(
 
   def expand(self, pvalue):
     return pvalue.pipeline | Read(self._source)
+
+
+class WriteToVcf(PTransform):
+  """A PTransform for writing to VCF files."""
+
+  def __init__(self,
+               file_path_prefix,
+               file_name_suffix='',
+               num_shards=0,
+               shard_name_template=None,
```

Comment:
I think this is not necessary and confusing to have as option (it only supports two options and one of them is empty string that is only applicable to 1 shard).